cmd/ilm-add.go

```diff
@@ -91,8 +98,14 @@ var ilmAddFlags = []cli.Flag{
 		Usage: "the number of days to transition",
 	},
 	cli.StringFlag{
-		Name:  "storage-class",
-		Usage: "storage class for current version to transition into. MinIO supports any warm tier configured via `mc-admin-tier-add`",
+		Name:   "storage-class",
+		Usage:  "storage class for current version to transition into. MinIO supports tiers configured via `mc-admin-tier-add`",
+		Hidden: true,
+	},
+	cli.StringFlag{
+		Name:   "tier",
+		Usage:  "remote tier where current versions will transition to",
+		Hidden: true,
```

Comment:
Are you sure this needs to be a hidden flag ?

Reply:
Good catch! Thanks.